pkg/operator/ceph/csi/spec.go

```diff
@@ -367,6 +367,44 @@ func startDrivers(namespace string, clientset kubernetes.Interface, ver *version
 			return errors.Wrap(err, "failed to load cephfs plugin service template")
 		}
 	}
+
+	pv, _ := clientset.CoreV1().PersistentVolumes().List(metav1.ListOptions{})
+	drivers := []string{CephFSDriverName, RBDDriverName}
+	driversPV := false
+	for _, p := range pv.Items {
+		if p.Spec.CSI == nil {
+			continue
+		}
+		if p.Spec.CSI.VolumeAttributes["clusterID"] == tp.Namespace {
+			for _, d := range drivers {
+				if d == p.Spec.CSI.Driver {
+					driversPV = true
```

Comment:
do we need to check with the driver to disable? with the current code, we are not disabling the RBD driver if cephfs PVC is found.

Reply:
Now, we are not checking for PVs so, we are good.